core/bridges/orm.go

```diff
@@ -22,6 +24,9 @@ type ORM interface {
 	CreateBridgeType(bt *BridgeType) error
 	UpdateBridgeType(bt *BridgeType, btr *BridgeTypeRequest) error
 
+	GetLastGoodResponse(dotId string, specId int32, maxElapsed time.Duration) ([]byte, error)
```

Comment:
A minor thing: everywhere else in the code you use term "cache", may these two methods should speak the same?
e.g. `GetCachedBridgeResponse`